app/webpack/webpack.common.config.js

```diff
@@ -0,0 +1,56 @@
+const path = require("path");
+const webpack = require("webpack");
+const CaseSensitivePathsPlugin = require("case-sensitive-paths-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  context: path.resolve(__dirname, "../src"),
+  entry: "./index.js",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "../dist")
+  },
+  resolve: {
+    extensions: [".tsx", ".jsx", ".ts", ".js"]
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: "ts-loader",
+        exclude: /node-modules/
+      },
+      {
+        test: /\.jsx?$/,
+        exclude: /(node_modules|bower_components)/,
+        loader: "babel-loader",
+        query: {
+          cwd: __dirname,
+        }
+      },
+      {
+        test: /\.jsx$/,
```

Comment:
[Optional] why eslint is checking jsx files only?
additionally, i'd recommend to use some preset (for example https://www.npmjs.com/package/eslint-config-airbnb)